db/sstables-format-selector.cc

```diff
@@ -28,9 +27,8 @@ void feature_enabled_listener::on_enabled() {
     }
 }
 
-sstables_format_selector::sstables_format_selector(gms::gossiper& g, sharded<gms::feature_service>& f, sharded<replica::database>& db)
-    : _gossiper(g)
-    , _features(f)
+sstables_format_selector::sstables_format_selector(sharded<gms::feature_service>& f, sharded<replica::database>& db)
```

Comment:
I'm not how to date on how this sstables format selector thing works, so I have perhaps a stupid question:
Can a node be configured to select an older format (e.g., mc) even though other formats are also supported? If this is the case, don't we need to update our gossiped state to say that this manually-selected format is the only one that this node agrees to support?